src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -115,6 +115,10 @@ public function theAdministratorTryToSeeTheSummaryOfCart(?string $tokenValue): v
      */
     public function iAddThisProductToTheCart(ProductInterface $product, ?string $tokenValue): void
     {
+        if (!$this->sharedStorage->has('cart_token')) {
+            $this->pickupCart();
+        }
+
```

Comment:
Should be removed, this logic is already in `putProductToCart()` method if I see correctly, and what is more, you have `$tokenValue` which is a value taken from `sharedStorage`